src/server/game/World.cpp

```diff
@@ -1372,6 +1376,62 @@ void World::LoadConfigSettings(bool reload)
     m_configs[CONFIG_CACHE_DATA_QUERIES] = sConfigMgr->GetBoolDefault("CacheDataQueries", true);
 }
 
+/// Get Server Patch
+char* const World::GetPatchName() const
```

Comment:
Not really important but I'd like we keep using std::string instead of char* whenever possible